src/theme/themes/_index.scss

```diff
@@ -1,6 +1,6 @@
 @forward "./default.scss";
 @forward "./professional.scss";
 @forward "./pfr.scss";
-@forward "./plh_facilitator_mx.scss";
+@forward "./plh_facilitator_mx/index";
```

Comment:
nit(blocking) As we already have a `plh_facilitator_mx` theme, which you haven't removed (`src/theme/themes/plh_facilitator_mx.scss`), this is confusing. I think the new theme should probably have a new name, so that the current production deployment can still use the old one for now.

Maybe the new one should should be called `plh_facilitator_mx_2`? @chrismclarke, any quick suggestions?